components/draw/SplitRouteMode.svelte

```diff
@@ -219,7 +219,7 @@
     // TODO Is there a cheaper way to do this?
     let start = lineFeature.geometry.coordinates[0];
     let sliced = lineSlice(start, pointFeature, lineFeature);
-    return length(sliced.geometry, { units: "kilometers" }) * 1000.0;
+    return length(sliced, { units: "kilometers" }) * 1000.0;
```

Comment:
Apparently the implementation is flexible about taking `Feature<LineString>` or its geometry, but the type checker got angry otherwise